src/platform/events/base.ts

```diff
@@ -1,31 +1,63 @@
+import { EntityType } from '../entity'
 import {
   BidAcceptedEvent,
   BidReceivedEvent,
+  CollectionCreatedEvent,
   ItemSoldEvent,
   RentalEndedEvent,
   RentalStartedEvent,
   RoyaltiesEarnedEvent
 } from './blockchain'
+import { CatalystDeploymentEvent } from './catalyst'
+import { MoveToParcelEvent } from './client'
 
-export enum EventType {
-  BID_ACCEPTED = 'bid-accepted',
-  BID_RECEIVED = 'bid-received',
-  ITEM_SOLD = 'item-sold',
-  RENTAL_ENDED = 'land-rental-ended',
-  RENTAL_STARTED = 'land-rental-started',
-  ROYALTIES_EARNED = 'royalties-earned'
+export namespace Events {
+  export enum Type {
+    BLOCKCHAIN = 'blockchain',
+    CATALYST_DEPLOYMENT = 'catalyst-deployment',
+    CLIENT = 'client'
+  }
+
+  export namespace SubType {
+    export enum Blockchain {
+      BID_ACCEPTED = 'bid-accepted',
+      BID_RECEIVED = 'bid-received',
+      ITEM_SOLD = 'item-sold',
+      RENTAL_ENDED = 'land-rental-ended',
+      RENTAL_STARTED = 'land-rental-started',
+      ROYALTIES_EARNED = 'royalties-earned',
+      COLLECTION_CREATED = 'collection-created'
+    }
+
+    export enum CatalystDeployment {
```

Comment:
There's already `EntityType` enum for this

Reply:
Yeah, I was using that type but then I realized that it can bring some confusion when creating an Event and using `EntityType` on them, even when the events are related to entities.

Ended up being something like:
```javascript
type: EventType.CATALYST_DEPLOYMENT
subType: EntityType.Store
```

and with this change it looks like:
```javascript
type: Events.Type.CATALYST_DEPLOYMENT
subType: Events.SubType.CatalystDeployment.Store
```


WDYT? I know now we have two places to update this when adding a new entity, but including that entity onto our event-driven architecture can be a separated step (making explicit the intention). Yet, I'm open to revert this if you have a strong opinion